### PR TITLE
OCP - Exposing z/OS Connect Designer through Wazi DevSpaces Workspace

### PR DESCRIPTION
**Pull request by midhun6989**
Changes corresponding to the [issue #69](https://github.com/IBM/zmodstack-deploy/issues/69).

### CODE REVIEW

File: .github/workflows/azure-deploy.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
     - name: Run Ansible Playbooks
       run: |
-        ansible-galaxy collection install azure.azcollection --force
+        ansible-galaxy collection install azure.azcollection:1.19.0 --force
```
Comment (ivandov) [inline .github/workflows/azure-deploy.yml:31]: Is there a reason we need to force a specific version? Shouldn't we always just grab the latest?

File: aws/cloudformation/cluster.yml [hunk 2]
```diff
@@ -560,7 +560,7 @@ Resources:
             mkdir -p $GIT_CLONE_DIR
 
             # Clone git repo containing automation scripts for AWS
-            git clone --no-checkout -b release-2023.4.1 https://github.com/IBM/zmodstack-deploy.git $GIT_CLONE_DIR
+            git clone --no-checkout -b issue-69 https://github.com/IBM/zmodstack-deploy.git $GIT_CLONE_DIR
```
Comment (ivandov) [inline aws/cloudformation/cluster.yml:563]: This needs to be set back to `release-2023.4.1`

File: aws/cloudformation/main.yml [hunk 3]
```diff
@@ -303,7 +303,7 @@ Resources:
     Type: AWS::CloudFormation::Stack
     Condition: AZValidation
     Properties:
-      TemplateURL: https://z-mod-stack-aws-dev.s3.us-east-2.amazonaws.com/branches/release-2023.4.1/cloudformation/vpc.yml
+      TemplateURL: https://z-mod-stack-aws-dev.s3.us-east-2.amazonaws.com/branches/issue-69/cloudformation/vpc.yml
```
Comment (ivandov) [inline aws/cloudformation/main.yml:306]: Set back to `release-2023.4.1`

File: aws/cloudformation/main.yml [hunk 4]
```diff
@@ -331,7 +331,7 @@ Resources:
     Condition: AZValidation
     Properties:
       # TemplateURL: https://ibm-zmodstack-public.s3.amazonaws.com/cloudformation/cluster.yml
-      TemplateURL: https://z-mod-stack-aws-dev.s3.us-east-2.amazonaws.com/branches/release-2023.4.1/cloudformation/cluster.yml
+      TemplateURL: https://z-mod-stack-aws-dev.s3.us-east-2.amazonaws.com/branches/issue-69/cloudformation/cluster.yml
```
Comment (ivandov) [inline aws/cloudformation/main.yml:334]: Set back to `release-2023.4.1`

**Review by ivandov — CHANGES_REQUESTED** [review]
Looks good, but need to fix some static references.

File: .github/workflows/azure-deploy.yml (hunk 1 above)
Comment (midhun6989) [reply .github/workflows/azure-deploy.yml:31]: The latest version (2.0.0) which got released 3 weeks ago is throwing errors when used in GitHub Actions. So used the previous version (1.19.0).
https://github.com/ansible-collections/azure/releases

**Review by ivandov — APPROVED** [review]
Using hardcoded values and will replace in `dev` branch before merge to `release-2023.4.1`

**Pull request merged**